README.md

```diff
@@ -1 +1,193 @@
+  <li><a href="https:///">Javascript</a></li>
+  <li><a href="https:///">ES6</a></li>
+  <li><a href="https:///">Webpack</a></li>
+  <li><a href="https:///">API</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Webpack Implementation]**
+- **[Css Loaders]**
+- **[Live server implementation]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link] [coming zoom]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Clone the repository using git, and open it with visual studio code.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  cd "your-folder"
+  git clone [https://github.com/joramot/Leaderboard_JoseRM.git]
+
+  git clone [git@github.com:https://github.com/joramot/Leaderboard_JoseRM.git]
+
+
+### Usage
+
+To open the project, use yout prefered explorer and open the index.html file 
+
+
+### Deployment
+
+You can deploy this project using visual studio code and Figma.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 ** Jose RM **
+
+- Project presentation: comming soon!!!
+
+- GitHub: https://github.com/joramot
+- Twitter: https://twitter.com/el_rulis
+- LinkedIn: https://linkedin.com/in/jose-RM
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Sort the scores from max to min and reverse.
+- [ ] Add player alias column.
+- [ ] add score awards column
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome! 
+Feel free to check the [issues page] (https://github.com/joramot/Leaderboard_JoseRM/issues)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can contact the author througt the above meddia links or by email to mojica.raul@gmail.com
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse and all the persons that got in touch with me while we were working on this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ❓ FAQ <a name="faq"></a>
+
+- Is it difficult?
+
+  - If you have knowledge about github basic commands, installing npm extension in visual studio code, figma, HTML and CSS it is easier, in other way, it can be little complicated
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] MIT License link is invalid. Please ensure to include the appropriate license file/update link